resources/skins/nudus-dark/qt_style.qss

```diff
@@ -0,0 +1,33 @@
+QTextEdit { /* TODO: Is this used to be what QPlainTextEdit is right now? But highlight works fine there presently. Should be removed? */
+    selection-background-color: yellow;
+}
+
+QListWidget,
+QScrollArea {
+    border: 1px solid palette(dark);
+}
+
+
+QPlainTextEdit:focus {
+    border: 1px solid palette(highlight);
+}
+
+QToolTip {
+    background-color: palette(window);
+    border: 1px solid palette(dark);
+    border-radius: 2px;
+}
+
+/* TODO: Fine for now, may be improved in future */
+QProgressBar {
+    background-color: palette(highlight);
+    color: palette(window);
+}
+
+QStatusBar::item { /* TODO: should this be removed? I don't see any changes in status bar */
+    border: none;
```

Comment:
@martinrotter And what about this?

Reply:
Any comment?

Reply:
I remember that it had some offect when other Qt styles were used - some styles made statusbar crazily thick, this css made it a bit smaller.

Reply:
I see. Will test.

Reply:
But it was long time ago, I believe it was "CDE" or "gnome" style which was used in Qt 4 era.

https://aur.archlinux.org/packages/qt5-styleplugins/

Reply:
> in Qt 4 era.

Oh. Well, I'll still see if there would be any regressions.

Thank you.

Reply:
Still can see problems with it in the most recent version of `qt5-styleplugins` (2017), with styles such as _motif_, _gtk 2_, _cde_, perhaps other.

Will add a comment explaining it.